pkg/controller/cloud/pvlcontroller.go

```diff
@@ -44,8 +44,6 @@ import (
 	volumeutil "k8s.io/kubernetes/pkg/volume/util"
 )
 
-const initializerName = "pvlabel.kubernetes.io"
```

Comment:
Still working through alternatives to this. Will loop back when we come up with something, hopefully within the next few days

Reply:
Spoke with @dims offline. Given that there are very few users actually using Initializers for PV labeling we're just going to port any remaining implementations of the PV labeler (OpenStack) into the in-tree PVL admission controller. This should buy us some time until a better solution that doesn't use Initializers is available out-of-tree. PR to update PVL admission controller coming soon.

Reply:
we will have to poll folks on sig-openstack slack and email, but at least we will have an alternative to recommend.